composer.json

```diff
@@ -4,6 +4,7 @@
     "description": "The best of Drupal, curated by Acquia",
     "license": "GPL-2.0-or-later",
     "require": {
+        "ext-dom": "*",
```

Comment:
Why did we add this?

Reply:
Because I noticed there's code that uses the DOM extension. (PhpStorm helpfully highlights any code that uses a PHP extension that's not required in `composer.json`.)